src/misc/simulation/fscommon/fsdirectories.cpp

```diff
@@ -141,6 +171,20 @@ namespace swift::misc::simulation::fscommon
         return dir;
     }
 
+    static QString msfs2024DirImpl()
+    {
+        const QStringList locations = QStandardPaths::standardLocations(QStandardPaths::GenericDataLocation);
+        for (const QString &path : locations)
+        {
+            const QString msfs2024Package = CFileUtils::appendFilePaths(CFileUtils::appendFilePaths(path, "Packages"),
+                                                                        "Microsoft.Limitless_8wekyb3d8bbwe");
+            const QDir d(msfs2024Package);
+            if (!d.exists()) { continue; }
+            return msfs2024Package;
+        }
+        return {};
+    }
+
```

Comment:
Is this intended to be in this PR?

Reply:
not really necessary. I had built the fix on the msfs2024_native branch and forgot to remove it for this version. So it will come into effect soon. I can still remove it if you think that's better

Reply:
Fine for me to leave it here